ern-container-gen-android/src/AndroidGenerator.ts

```diff
@@ -707,7 +691,49 @@ You should replace "${annotationProcessorPrefix}:${dependency}" with "annotation
           'lib/src/main/java/com/walmartlabs/ern/container/plugins',
         );
         shell.cp(pathToPluginConfigHook, pathToCopyPluginConfigHookTo);
+
+        this.updatePluginsToAndroidxImports(
+          rnVersion,
+          plugin,
+          pathToCopyPluginConfigHookTo,
+        );
+      }
+    }
+  }
+
+  /**
+   * Modify Plugins to support Androidx
```

Comment:
Minor: Please use correct caps in docs: `Androidx` -> `AndroidX` - With the refactored generic version, the subject line here could be something like `Convert files in a directory from support library to AndroidX`.